telco-ran/configuration/source-crs/node-tuning-operator/TunedPerformancePatch.yaml

```diff
@@ -1,32 +1,42 @@
 apiVersion: tuned.openshift.io/v1
 kind: Tuned
 metadata:
-  name: performance-patch
+  name: ran-du-performance
   namespace: openshift-cluster-node-tuning-operator
   annotations:
     ran.openshift.io/ztp-deploy-wave: "10"
 spec:
   profile:
-    - name: performance-patch
+    - name: ran-du-performance
       # Please note:
       # - The 'include' line must match the associated PerformanceProfile name, following below pattern
       #   include=openshift-node-performance-${PerformanceProfile.metadata.name}
-      # When using the standard (non-realtime) kernel, disable timer migration by adding the following in the data section:
-      # [sysctl]
-      # kernel.timer_migration=0
       data: |
         [main]
-        summary=Configuration changes profile inherited from performance created tuned
-        include=openshift-node-performance-openshift-node-performance-profile
+        summary=Top-level ran-du performance tuning overrides
+        include=openshift-node-performance-openshift-node-performance-profile,ran-du-performance-architecture-common
+    - name: ran-du-performance-architecture-common
+      data: |
+        [main]
+        summary=Common cross-architecture performance tuning which also includes architecture-specific profiles
+        include=performance-patch-${f:lscpu_check:Vendor ID\:\s*GenuineIntel:intel:Vendor ID\:\s*AuthenticAMD:amd:Architecture\:\s*aarch64:arm}-${f:lscpu_check:Architecture\:\s*x86_64:x86:Architecture\:\s*aarch64:aarch64}
```

Comment:
I think I see a potential problem here - We want to include the ice driver scheduling section for both intel and amd (ie for all x86_64).

Maybe it's better to just have a simple x86_64/aarch64 determination at this high level, and the child policies could further differentiate if needed... But for today we only need this simpler x86_64/aarch difference.